docs/docs/render-props.md

```diff
@@ -0,0 +1,226 @@
+---
+id: render-props
+title: Render Props
+permalink: docs/render-props.html
+---
+
+The term ["render prop"](https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce) refers to an advanced technique for sharing code between React components using a prop whose value is a function instead of a resorting to using a [higher-order component](/react/docs/higher-order-components.html) (HOC).
+
+## Use Render Props for Cross-Cutting Concerns
+
+A render prop is a simple, elegant way to resolve cross-cutting concerns in your React app without [the caveats of higher-order components](/react/docs/higher-order-components.html#caveats).
```

Comment:
Stop selling.

Reply:
I think the sell is good, because of the link there I was able to understand when to use a HoC and when to use a render prop. After reading and understanding I have use cases for both.